planner/core/testdata/integration_suite_out.json

```diff
@@ -203,6 +203,25 @@
           "├─IndexRangeScan_6(Build) 1.00 cop[tikv] table:t, index:b(b) range:[2,2], keep order:false, stats:pseudo",
           "└─TableRowIDScan_7(Probe) 2.00 cop[tikv] table:t keep order:false, stats:pseudo"
         ]
+      },
+      {
+        "SQL": "explain select /*+ USE_INDEX_MERGE(t, primary) */ * from t where 1 or t.c",
```

Comment:
seems we can fold this `1 or t.c` to `t.c`, thus avoid a table full scan?

Reply:
`1 or t.c` is equivalent to `1`. It's a test for index merge to avoid panic.